src/Microsoft.AspNetCore.TestHost/TestServer.cs

```diff
@@ -27,6 +27,19 @@ public TestServer(IWebHostBuilder builder)
             _hostInstance = host;
         }
 
+        public TestServer(IWebHostBuilder builder, IFeatureCollection featureCollection)
```

Comment:
If you're going to pass in the collection then just directly assign it to IServer.Features. No need to copy it.